ShippingApp/src/main/java/com/javaschool/queue/ConfigRabbit.java

```diff
@@ -0,0 +1,37 @@
+package com.javaschool.queue;
+
+import org.springframework.amqp.core.Binding;
+import org.springframework.amqp.core.BindingBuilder;
+import org.springframework.amqp.core.Queue;
+import org.springframework.amqp.core.TopicExchange;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ConfigRabbit {
+
+    @Value("${config-rabbit.queue}")
+    public String QUEUE_SHIPPING;
```

Comment:
ALL_CAPS convention is only for constants

Reply:
The names of the variables were modified.